modules/node-shared/src/main/scala/org/tessellation/node/shared/domain/seedlist/SnapshotOrdinalTimeline.scala

```diff
@@ -0,0 +1,31 @@
+package org.tessellation.node.shared.domain.seedlist
+
+import cats.data.NonEmptyList
+import cats.syntax.all._
+
+import org.tessellation.node.shared.domain.seedlist.Range.{mergeOverlaps, parseInput}
+
+import fs2.data.csv.{CellDecoder, DecoderError}
+
+case class SnapshotOrdinalTimeline(values: NonEmptyList[Range])
```

Comment:
I think it may be a `newtype`, what do you think?

Reply:
Changed to a `newtype`.